splink/files/chart_defs/tf_adjustment_chart.json

```diff
@@ -182,8 +179,7 @@
                             "field": "log2_bf_final"
                         },
                         "y2": {
-                            "type": "quantitative",
-                            "field": "log2_bf"
```

Comment:
@samnlindsay Similarly, the 'field' here has a validation error.  I think it can be omitted without loss